### PR TITLE
K8s engines on-prem spike results

### PR DESCRIPTION
**Pull request by sk4zuzu**

### CODE REVIEW

**Review by erzetpe — APPROVED** [review]
LGTM.

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+kube-system     pod/metrics-server-77956db857-pdklq            1/1     Running     0          38s
+kube-system     pod/rke-coredns-addon-deploy-job-2h25s         0/1     Completed   0          46s
+kube-system     pod/rke-ingress-controller-deploy-job-9882t    0/1     Completed   0          36s
+kube-system     pod/rke-metrics-addon-deploy-job-l47wt         0/1     Completed   0          41s
+kube-system     pod/rke-network-plugin-deploy-job-xzrwp        0/1     Completed   0          56s
+```
+
+### 3.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated (but easy to implement)
```
Comment (mkyc) [inline docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md:66]: can you please provide links to resources explaining those two things?

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+kube-system   pod/nodelocaldns-fmb5b                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-kjx77                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-lr4cf                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-pvlwc                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-tjwfr                           1/1     Running   0          51s
+```
+
+### 4.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated
```
Comment (mkyc) [inline docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md:181]: can you please provide links to resources explaining those two things?

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+kube-system   pod/node-local-dns-7h5m8                          1/1     Running   0          64s
+kube-system   pod/node-local-dns-7tsvf                          1/1     Running   0          82s
+kube-system   pod/node-local-dns-c58zf                          1/1     Running   0          82s
+kube-system   pod/node-local-dns-dvnbb                          1/1     Running   0          57s
+kube-system   pod/node-local-dns-vsjpb                          1/1     Running   0          82s
+```
+
+### 5.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated (but provides at least a script to download docker images)
```
Comment (mkyc) [inline docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md:299]: can you please provide links to resources explaining those two things?

**Review by mkyc — CHANGES_REQUESTED** [review]
Small clarifications required.

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md (hunk 1 above)
Comment (sk4zuzu) [reply docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md]: added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md (hunk 2 above)
Comment (sk4zuzu) [reply docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md]: added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation

File: docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md (hunk 3 above)
Comment (sk4zuzu) [reply docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md]: added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation

**Review by mkyc — APPROVED** [review]
LGTM

**Review by erzetpe — APPROVED** [review]

**Pull request merged**